internal/pkg/db/mongo/models/deviceprofile.go

```diff
@@ -91,8 +91,8 @@ type DeviceProfile struct {
 	Model           string            `bson:"model"`
 	Labels          []string          `bson:"labels"`
 	DeviceResources []DeviceResource  `bson:"deviceResources"`
-	Resources       []ProfileResource `bson:"resources"`
-	Commands        []mgo.DBRef       `bson:"commands"`
+	DeviceCommands  []ProfileResource `bson:"resources"`
+	CoreCommands    []mgo.DBRef       `bson:"commands"`
```

Comment:
Bson names shouldn't be changed?

Reply:
There have been many important changes in DeviceProfile in this release cycle which force the Mongo database to be flushed, so I think should be more clear if all names are aligned.
But anyway, it's fine for me to leave as it is.